docs/Tutorial_DiscordMERLIN.ipynb

```diff
@@ -0,0 +1,1714 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    def _discords(T, m, min_dist, n_discords=1, decay = None, s = None, include = None, finite=False):
</pre><p>Instead of <code>n_discords</code> let's call it <code>k</code> to be consistent with <code>top-k</code> discords   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>